### DISCUSSION TITLE
plaso timeline files

### DISCUSSION THREAD

**Discussion by tamermh** [Security-Onion-Solutions/securityonion#4409] [Unsupported Versions] [answered]
Hi,

how can plaso files ne ingested in security onion? 

Thanks

**Comment by weslambert** [comment, marked as answer]
You can move the files to a Security Onion server and import them there via FIlebeat (or install Filebeat on the server housing the files and read them from there), then use the Elastic Ingest Node CSV processor to parse these.  Alternatively, you could use the Logstash file input, or emulate the ES bulk upload API, and send to Logstash over the network (I wouldn't recommend sending directly to ES if you don't have to open the port externally). I'll see if I can put together an example soon.
